catroid/src/org/catrobat/catroid/content/bricks/StopScriptBrick.java

```diff
@@ -0,0 +1,179 @@
+//		return NO_RESOURCES;
+//	}
+
+	@Override
+	public View getViewWithAlpha(int alphaValue) {
+		if (view != null) {
+			View layout = view.findViewById(R.id.brick_stop_script_layout);
+			Drawable background = layout.getBackground();
+			background.setAlpha(alphaValue);
+
+			Spinner spinner = (Spinner) view.findViewById(R.id.brick_stop_script_spinner);
+			spinner.getBackground().setAlpha(alphaValue);
+
+			TextView textStopScript = (TextView) view.findViewById(R.id.brick_stop_script_label);
+//			ColorStateList color = textStopScript.getTextColors().withAlpha(alphaValue);
+			textStopScript.setTextColor(textStopScript.getTextColors().withAlpha(alphaValue));
+
+//			if(adapterView != null) {
+//				((TextView) adapterView.getChildAt(0)).setTextColor(color);
+//			}
+
+			this.alphaValue = alphaValue;
+		}
+		return view;
+	}
+
+	@Override
+	public View getView(Context context, int brickId, BaseAdapter baseAdapter) {
+		if (animationState) {
+			return view;
+		}
+		view = View.inflate(context, R.layout.brick_stop_script, null);
+		view = getViewWithAlpha(alphaValue);
+		setCheckboxView(R.id.brick_stop_script_checkbox);
+
+		final Brick brickInstance = this;
+		checkbox.setOnCheckedChangeListener(new OnCheckedChangeListener() {
+			@Override
+			public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
+				checked = isChecked;
+				adapter.handleCheck(brickInstance, isChecked);
+			}
+		});
+
+		Spinner spinner = (Spinner) view.findViewById(R.id.brick_stop_script_spinner);
+		if(!(checkbox.getVisibility() == view.VISIBLE)) {
+			spinner.setClickable(true);
+			spinner.setEnabled(true);
+		} else {
+			spinner.setClickable(false);
+			spinner.setEnabled(false);
+		}
+
+		ArrayAdapter<String> spinnerAdapter = createArrayAdapter(context);
+		spinner.setAdapter(spinnerAdapter);
+		spinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+			@Override
+			public void onItemSelected(AdapterView<?> adapterView, View view, int position, long l) {
+				spinnerSelectionId = position;
+			}
+
+			@Override
+			public void onNothingSelected(AdapterView<?> parent) {
+			}
+		});
+
+		spinner.setSelection(spinnerSelectionId);
+
+		return view;
+	}
+
+	@Override
+	public View getPrototypeView(Context context) {
+		View prototypeView = View.inflate(context, R.layout.brick_stop_script, null);
+
+		Spinner spinner = (Spinner) prototypeView.findViewById(R.id.brick_stop_script_spinner);
+		spinner.setEnabled(false);
+		spinner.setFocusable(false);
+		spinner.setFocusableInTouchMode(false);
+
+		ArrayAdapter<String> spinnerAdapter = createArrayAdapter(context);
+		spinner.setAdapter(spinnerAdapter);
+		spinner.setSelection(spinnerSelectionId);
+
+		return prototypeView;
+	}
+
+	@Override
+	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
+		sequence.addAction(sprite.getActionFactory().createStopScriptAction(spinnerSelectionId, sequence));
+		return null;
```

Comment:
Could you return an empty list here too? See [this](https://github.com/Catrobat/Catroid/pull/1810#discussion_r72038248)